example/javascriptlib/testing/1-test-suite/build.mill

```diff
@@ -0,0 +1,22 @@
+package build
+
+import mill._, javascriptlib._, javascriptlib.testmods._
+
+object bar extends JestModule
+
+object foo extends MochaModule
+
```

Comment:
I know the original ticket asked for 2 test frameworks `mocha` and `jest`, but I'd like to ask you to add two more: `Vitest` and `Jasmine`, which are the third and fourth most popular Javascript testing frameworks according to the Jetbrains developer survey https://www.jetbrains.com/lp/devecosystem-2023/javascript/

Reply:
will open a  new pr for Vitest and jasmine